lib/date.js

```diff
@@ -175,13 +175,38 @@ function isoDateFromDateInput (object, namePrefix) {
   }
 }
 
+/**
+ * Convert a number into name of corresponding month.
+ *
+ * @see {@link https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates}
+ *
+ * @example
+ * month(3) // March
+ * month(3, 'truncate') // Mar
+ *
+ * @param {number} number - Value to convert
+ * @param {boolean|string} [format=false] - Truncate name
+ * @returns {string} Month name
+ */
+function month (number, format = false) {
+  const monthIndex = number - 1
+  const date = new Date(2012, monthIndex)
+  const truncate = format === 'truncate'
+
+  return new Intl.DateTimeFormat('en-GB', {
```

Comment:
Nice - didn’t know about this API! Any idea which version of Nodejs it works in from?

Reply:
v13.0.0 for much of the API.